src/consensus/pbft/libbyz/Pre_prepare.h

```diff
@@ -25,6 +25,7 @@ struct Pre_prepare_rep : public Message_rep
   View view;
   Seqno seqno;
   std::array<uint8_t, MERKLE_ROOT_SIZE> merkle_root;
+  std::array<uint8_t, MERKLE_ROOT_SIZE> r_merkle_root;
```

Comment:
Can you please give this a more descriptive name

Reply:
going for `full_state_merkle_root` and `replicated_state_merkle_root` here. In history the trees will be `full_state_tree` and `replicated_state_tree` and the respective functions `get_full_state_root()` and `get_replicated_state_root()` instead of `get_root()` and `get_r_root()`